packages/databricks-vscode/src/configuration/ui/ConfigurationDataProvider.ts

```diff
@@ -42,7 +42,7 @@ export class ConfigurationDataProvider
         private readonly configModel: ConfigModel
     ) {
         this.disposables.push(
-            this.bundleProjectManager.onDidChangeStatus(() => {
+            this.bundleProjectManager.onDidChangeStatus(async () => {
```

Comment:
Need this because it is easier to handle always async functions in the event wrapper.